server/src/services/deviceService.ts

```diff
@@ -63,31 +66,25 @@ class DeviceService implements EventObserver {
         });
     }
 
-    async getDeviceById(id: string) {
+    /**
+     * Get a device by its id
+     * @param id Id of the device
+     * @returns the device with the correct id, or undefined
+     */
+    async getDeviceById(id: string): Promise<Device | undefined> {
         const devices: Device[] = await this.getDevices();
         return devices.find((d: Device) => d.id === id);
     }
 
-    async pairdevice(): Promise<any> {
-        try {
-            await httpForwarder.post<any>('/api/services/zwave/add_node', null);
-        } catch (err) {
-            console.log(err);
-        }
-        try {
-            await socketForwarder.forward({
-                type: 'call_service',
-                domain: 'mqtt',
-                service: 'publish',
-                service_data: {
-                    topic: 'zigbee2mqtt/bridge/request/permit_join',
-                    // payload_template: '"{"value": true}"',
-                    payload_template: 'true',
-                },
-            });
-        } catch (err) {
-            console.log(err);
-        }
+    /**
+     * Pair a new device
+     */
+    async pairDevice(): Promise<void> {
```

Comment:
why delete try catch ? if zwave request fail zigbee will not enter into pairing device